endpoints/v2/test/test_blob.py

```diff
@@ -109,8 +116,9 @@ def test_blob_mounting(mount_digest, source_repo, username, expect_success, clie
     params = {
         "repository": "devtable/building",
         "mount": mount_digest,
-        "from": source_repo,
```

Comment:
should we cover both cases here? i.e. one test ensuring the blob is mounted correctly with `from`, and another ensuring it will also work without it.

Reply:
We are. `include_from_param` parameterize the test to include or not `from`.
See https://github.com/quay/quay/pull/899/files#diff-6015d1dddcac40a4d4acd470fe07b31e94e8ec22bf157d361b5ad67c136ec133R80

Reply:
ooh that flew right over my head, neat!